types/unified-tests.ts

```diff
@@ -30,7 +30,7 @@ const settings = {
 interface ExamplePluginSettings {
   example: string
 }
-const typedPlugin: Plugin<ExamplePluginSettings> = function() {}
+const typedPlugin: Plugin<[ExamplePluginSettings?]> = function() {}
```

Comment:
Note that plugin authors, choosing to add an explicit type definition would need to switch to defining settings in a tuple.
Plugin authors leveraging inferencing would see no difference.